packages/bvaughn-architecture-demo/components/sources/PointPanelTimeline.tsx

```diff
@@ -86,7 +86,7 @@ export default function PointPanelTimeline({
 
   const label =
     currentHitPointIndex !== null
-      ? `${currentHitPointIndex + 1} / ${hitPoints.length}`
+      ? `${currentHitPointIndex + 1}/${hitPoints.length}`
```

Comment:
Huh. Was this a Prettier change? 🤔

Reply:
nope - the figma design had it and it looked better. In reality i think some padding is ideal, this is too little, before was too much.

Reply:
Oh, sorry. I misread this at first pass.